examples/reciprocal-events-timebank

```diff
@@ -0,0 +1,72 @@
+'@context':
+  '@vocab': https://w3id.org/valueflows#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  time: http://www.w3.org/2006/time#
+  time:inXSDDateTimeStamp:
+    '@type': xsd:dateTimeStamp
+  timebank: https://timebank.example
+  alice: https://alice.example
+  bob: https://bob.example
+
+'@graph':
+
+  - '@id': timebank:234234
+    '@type': EconomicResource
+    name: 'Alice credit account'
+
+  - '@id': timebank:657897
+    '@type': EconomicResource
+    name: 'Bob credit account'
+
+  - '@id': timebank:7dd207e3-8f11-4fe3-95f8-d02f6c180be3
+    '@type': EconomicEvent
+    provider: alice
+    recipient: bob
+    action: work
+    observedQuantity:
+      qudt:unit: Hours
+      qudt:numericValue: 5
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q203605 #carpentry
+    observedTime:
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-12-29T00:00:00-5:00
+
+  - '@id': timebank:7dd207e3-8f11-4fe3-95f8-d02f6c49084
+    '@type': EconomicEvent
+    provider: bob
+    recipient: alice
+    action: give
+    observedQuantity:
+      qudt:unit: Credits
+      qudt:numericValue: 5
+    affects: timebank:657897
+    observedTime:
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-12-30T00:00:00-5:00
+
+  - '@id': timebank:7dd207e3-8f11-4fe3-95f8-d02f6c180be3
+    '@type': EconomicEvent
+    provider: bob
+    recipient: alice
```

Comment:
If *give* event already have provider and recipient why do you also have it repeated on *receive* event?
I would find it clearer if either *give* or *receive* have provider and recipient but not both. Otherwise it appears like Bob provided 5 credits to Alice twice.

Reply:
Because `give` event is only in the scope of the provider and `receive` event is only in the scope of the receiver, so they wouldn't see the other one - not always, but often.

Reply:
I think there are several ways this can be modelled, and none of them are perfectly clean, they all have trade-offs.  At least, I haven't found the perfect one yet.